src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -38,6 +38,7 @@
     public static final String FORMAT_OUTPUT_FIELDS_CONFIG = "format.output.fields";
     public static final String FORMAT_OUTPUT_FIELDS_VALUE_ENCODING_CONFIG = "format.output.fields.value.encoding";
     public static final String FORMAT_OUTPUT_TYPE_CONFIG = "format.output.type";
+    public static final String FORMAT_OUTPUT_JSON_ENVELOPE_CONFIG = "format.output.json.envelope";
```

Comment:
Since we're making this feature not JSON specific, let's change the config name to `format.output.envelope` (+related places like doc strings and variable names).